crawl-ref/source/enum.h

```diff
@@ -370,6 +370,7 @@ enum ability_type
     ABIL_ASHENZARI_SCRYING = 1160,
     ABIL_ASHENZARI_TRANSFER_KNOWLEDGE,
     ABIL_ASHENZARI_END_TRANSFER,
+    ABIL_ASHENZARI_CURSE,
```

Comment:
is this ordering OK? I added it at the end for save-compat reasons, but it's out of order with the ability unlocks